main.sh

```diff
@@ -232,7 +224,7 @@ if [[ "${INPUT_TAR/all/${platform}}" == "${platform}" ]] || [[ "${INPUT_ZIP/all/
         cp "${target_dir}/${bin_exe}" "${tmpdir}/${archive}"/
     done
     for include in ${includes[@]+"${includes[@]}"}; do
-        cp -r "${include}" "${tmpdir}/${archive}"/
+        cp -r ${include} "${tmpdir}/${archive}"/ || true
```

Comment:
Why is `|| true` needed?

Reply:
Ah, this seems related to your comment in #35

> The include option is also a bit useless if it fails the action if it cant find a file, there may be platform dependent files such as libexample.so and example.dll but all builds fail when one of them doesn't match on one target matrix.

The current behavior is intentional, to avoid missing cases where needed files are missing.

Currently, the way to specify platform-specific files is [to use a matrix](https://github.com/openrr/urdf-viz/blob/8803d7d4b65bc91691d354d97250744c09cc0fe6/.github/workflows/release.yml#L51-L72), but it is okay to add an option to allow files that do not exist.